samples/net/sockets/echo/src/socket_echo.c

```diff
@@ -73,7 +73,7 @@ int main(void)
 		printf("Connection #%d from %s\n", counter++, addr_str);
 
 		while (1) {
-			char buf[128], *p;
+			static char buf[8192], *p;
```

Comment:
can you tell why in the commit message? (besides reducing recv() calls)
Because it may affect some targets with limited amount of ram.